utils.js

```diff
@@ -85,6 +89,17 @@ function ecsMeta(req, res, err) {
     if (port) meta.url.port = Number(port);
   }
 
+  if (req.ctx && isPlainObject(req.ctx)) {
```

Comment:
Not sure I understand the condition here. What `req.ctx` can ever be?

Reply:
There is always a chance that some service can use different implementations for context (like WeakMap, etc.).
So I meant for the logger to be resilient to that.